www/static/js/index.js

```diff
@@ -145,17 +145,46 @@ $(document).ready(function () {
 
     // Smooth scroll to anchor links
     $("a[href^='#']").on('click', function(e) {
-        if(this.hash) {
+
+        // scroll only if there is a hash in the href
+        var hash = this.hash;
+        if (hash) {
 
             // prevent default anchor click behavior
             e.preventDefault();
 
-            // store hash
-            var hash = this.hash;
+            // get the fragment without the "#" symbol because location.hash
+            // is returned with it
+            var targetName = targetName.slice(1);
```

Comment:
This line is a little confusing. Can you make the scoping of `targetName` clearer by renaming the variable?